sdk/tests/deqp/modules/shared/glsTextureTestUtil.js

```diff
@@ -1218,7 +1220,8 @@ glsTextureTestUtil.computeLodFromDerivates2D = function(mode, dudx, dvdx, dudy,
             throw new Error('Unrecognized mode:' + mode);
     }
 
-    return Math.log(p) * 1.44269504089;
+    // Native dEQP uses 32-bit numbers. So here 64-bit floating numbers should be transformed into 32-bit ones to ensure the correctness of the result.
+    return deMath.toFloat32(Math.log(p)) * deMath.INV_LOG_2_FLOAT32;
```

Comment:
Here, 64-bit floating numbers should be transformed into 32-bit ones to get the correct answer.